plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -57,6 +57,9 @@ class SlurmQueueConf(BaseQueueConf):
     # check the following for more info on slurm_max_num_timeout
     # https://github.com/facebookincubator/submitit/blob/master/docs/checkpointing.md
     max_num_timeout: int = 0
+    # Useful to add parameters which are not currently available in the plugin.
```

Comment:
what do you mean by "Useful to add parameters which are not currently available in the plugin"?
what happens when they become available in the plugin?

Reply:
If the plugin starts supporting all the slurm parameters, we do not need to have this variable :)

Reply:
ok. I guess we will cross that bridge later, but if for example support for email is added it's not clear what would be the behavior where the email is becoming supported with regard to existing user's configs.

Reply:
Thats a good point.